py-polars/src/cloud.rs

```diff
@@ -1,12 +1,14 @@
-use pyo3::exceptions::PyAssertionError;
+use pyo3::exceptions::PyValueError;
 use pyo3::prelude::*;
+use pyo3::types::PyBytes;
 
 use crate::PyLazyFrame;
 
 #[pyfunction]
-pub fn assert_cloud_eligible(lf: PyLazyFrame) -> PyResult<()> {
-    let plan = &lf.ldf.logical_plan;
-    polars::prelude::assert_cloud_eligible(plan)
-        .map_err(|e| PyAssertionError::new_err(e.to_string()))?;
-    Ok(())
+pub fn prepare_cloud_plan(lf: PyLazyFrame, uri: String, py: Python) -> PyResult<PyObject> {
+    let plan = lf.ldf.logical_plan;
+    let bytes = polars::prelude::prepare_cloud_plan(plan, uri)
+        .map_err(|e| PyValueError::new_err(e.to_string()))?;
```

Comment:
I think this should go to `PyPolarsErr` to give the proper polars exception that matches `PolarsError`.